src/game.cpp

```diff
@@ -9415,10 +9415,12 @@ bool game::walk_move( const tripoint &dest_loc )
         multiplier *= 3;
     }
 
-    const int mcost = m.combined_movecost( u.pos(), dest_loc, grabbed_vehicle, modifier ) * multiplier;
+    const int mcost = m.combined_movecost( u.pos(), dest_loc, grabbed_vehicle, modifier,
+                                           via_ramp ) * multiplier;
     if( grabbed_move( dest_loc - u.pos() ) ) {
         return true;
     } else if( mcost == 0 ) {
+        printf( "\t walk failed from 0 move cost\n" );
```

Comment:
Maybe print a debug message instead?

Reply:
nah, that's a debug message that I missed.  Fixed, thanks.